.script/checkThatTemplatesVersionWasChanged.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+var=$(echo $(git diff origin/master -U0))
+
+if [[ "$var" == *"version"* ]];
+then
+    echo "Changed the version of the template, nice!"
+    exit 0
+else
+    echo "Hey there, you did not change the version of the template, please update the version of the template you changed."
```

Comment:
just to make the comment a bit more clear, maybe use something like
"""
There are one or more templates that were modified and their version was not updated. Please update the version of the template(s) you changed.
"""